### ISSUE TITLE
Doesn’t work in background

### ISSUE THREAD

**Issue by AtifSayings** [quentin7b/android-location-tracker#30]
When app is in background or screen is locked, then after some seconds location updates stops.

**Comment by qk7b** [comment]
Maybe it is an android restriction, have you tried to put it in a Foregrond Service ?

**Comment by gautamsamyak** [comment]
yes, i have tried to put it in foreground but this is not worked
